app/models/project.rb

```diff
@@ -106,7 +106,7 @@ class Project < ActiveRecord::Base
   validates_length_of :homepage, maximum: 255
   validates_length_of :identifier, in: 1..IDENTIFIER_MAX_LENGTH
   # downcase letters, digits, dashes but not digits only
-  validates_format_of :identifier, with: /\A(?!\d+$)[a-z0-9\-_]*\z/,
+  validates_format_of :identifier, with: /\A[a-z][a-z0-9\-_]*/,
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.